recipe/build.sh

```diff
@@ -35,6 +35,7 @@ bash configure --prefix=$PREFIX \
                --with-hdf4=$PREFIX \
                --with-cfitsio=$PREFIX \
                --with-hdf5=$PREFIX \
+	       --with-tiledb=$PREFIX \
```

Comment:
It is a minor thing but can you fix the indentation here?

Reply:
fixed